lib/msal-react/package.json

```diff
@@ -55,5 +55,10 @@
     "tslib": "^2.0.0",
     "typescript": "^3.9.7"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "browserslist" : [
+    ">2%",
+    "not dead",
+    "last 2 years"
```

Comment:
It would be good for us to formally document this and make sure we are consistent in all the client side libs (namely msal browser and its wrapper libs)

Reply:
https://browserslist.dev/?q=bGFzdCAyIHZlcnNpb25zLCBub3QgZGVhZCwgPjIl

Reply:
last 2 versions will still pick up IE11 and include this polyfill

So choices basically are: 
1. Our browserslist supports ie11 and we live with the broken CSP
2. Our browserslist supports ie11 and we stop using tsdx (not totally opposed to this as it seems it's not well maintained and pretty opinionated but it does make many things easier...🤷‍♂️)
3. Our browserslist doesn't support ie11 but we tell people we are still supporting it and they need to provide the polyfills (which they need to do anyway even with choices 1 and 2)

Side note: I'm not really sure why this polyfill is included if we're not using the functions it's supposed to polyfill...I would have expected the bundler to be smarter than that. Or maybe we are using these functions and I don't see where...

Reply:
MSAL Browser already requires polyfills for IE11, so I think it is fine to require that for MSAL React.